docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Deploying to a DigitalOcean Droplet
+---
+
+This guide walks through how to deploy and host your Gatsby site on a [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/).
+
+DigitalOcean provides a cloud platform to deploy, manage, and scale applications of any size, removing infrastructure friction and providing predictability so developers and their teams can spend more time building better software.
+
+DigitalOcean's product droplets are scalable compute IaaS (Infrastructure as a Service) or a VPS (Virtual Private Server) on the cloud which has great reliability and scalability. They come with varied price ranges ideal for small apps to giant enterprise-level apps.
+
+They provide service to select from various Unix-based distributions and select your technology-based platform with preinstalled pre-requisites from the marketplace. This guide will walk through the specific options that work best for deploying a Gatsby site with DigitalOcean.
+
+A Droplet can be spun up in less than a minute for as little as \$5/month.
+
+## Prerequisites
+
+- A Gatsby site living in a Git repository (GitHub, GitLab, or any Git cloud)
+- A [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/) with a non-root user configured with sudo group ([example: Ubuntu 18.04](https://www.digitalocean.com/community/tutorials/how-to-create-a-new-sudo-enabled-user-on-ubuntu-18-04-quickstart))
+- A custom domain name for your Gatsby site to help with configuring HTTPS
+
+## How to deploy your Gatsby site to DigitalOcean
+
+### Install NodeJS, NPM and Gatsby-CLI onto your droplet
+
+Follow these instructions for installs on an Ubuntu droplet.
+
+1. Log in to your droplet as a non-root user.
+
+2. Install NodeJS
+
+   ```bash
+   sudo apt-get update
+   sudo apt-get install nodejs
+   ```
+
+3. Install npm
+
+   ```bash
+   sudo apt-get install npm
+   ```
+
+   To view the version of node and npm installed, run,
+
+   ```bash
+   node -v
+   npm -v
+   ```
+
+4. To install the latest stable release using the `n` package (Optional) (Recommended),
```

Comment:
```suggestion
4. To install the latest stable Node.js release using the `n` package (Required),
```
apt-get installed Node 8, and Gatsby now requires Node 10 or higher. So this should be a required step.